package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@acuris/eslint-config",
-  "version": "0.1.16",
+  "version": "0.1.17",
```

Comment:
I don't think you need to bump the version.  I think it is done when you publish it.  See
https://github.com/mergermarket/acuris-eslint/blob/master/README-INTERNAL.md

Reply:
Thanks Steve. Reverted changing version. Please take a look again.